app/lib/efile/nj/nj_state_wages.rb

```diff
@@ -0,0 +1,15 @@
+module Efile
+  module Nj
+    class NjStateWages
+      NO_WAGES = -1
+
+      def self.calculate_state_wages(intake)
```

Comment:
[dust] Is this not covered in the calculator class already?

Reply:
i moved the wages logic (and the age check logic) out of the calculator since it now needs to be used in two separate places-- let me know if that choice sounds ok

Reply:
Moving shared logic out into its own class seems fine to me to avoid rewriting logic in two different places.

Reply:
I was able to get rid of the age check util altogether but kept this wages one